packages/artifactor/test/contracts.js

```diff
@@ -230,20 +229,20 @@ describe("artifactor + require", () => {
           "Fallback should not have been triggered yet"
         );
         return example.sendTransaction({
-          value: web3.utils.toWei("1", "ether")
+          value: interfaceAdapter.utils.toWei("1", "ether")
```

Comment:
I think these need to go somewhere else. Use web3 for this for now. The point of the interface adapter is that it removes usage of the Web3.js interface. In places where it doesn't serve to sufficiently replace the Web3.js interface adapter, we should use either the shim or Web3.js itself